docs/tracing/getting-started.md

```diff
@@ -0,0 +1,157 @@
+title: Getting Started with Sematext Tracing
+description: Learn how to set up distributed tracing with OpenTelemetry and Sematext
+
+## Getting Started with Sematext Tracing
+
+Sematext Tracing helps you monitor and troubleshoot distributed applications by tracking requests as they flow through your services. Built on OpenTelemetry, it provides visibility into performance bottlenecks, error propagation, and service dependencies.
+
+## Prerequisites
+
+Before you begin, ensure you have:
+
+- A Sematext Cloud account ([sign up free](https://apps.sematext.com/ui/registration))
+- Access to your application's source code or deployment configuration
+- Ability to install the Sematext Agent on your infrastructure
+
+## How It Works
+
+Sematext Tracing works by:
+
+1. Instrumenting Your Application: Add OpenTelemetry libraries to your code to automatically capture traces
```

Comment:
Not necessarily "to your code", but to the deployment, as env vars, startup params, configs, and such, not as code modifications, right?